sambacc/ctdb.py

```diff
@@ -483,6 +483,11 @@ def _convert_tdb_file(tdb_path: str, dest_dir: str, pnn: int = 0) -> None:
     subprocess.check_call(list(cmd))
 
 
+def check_ok(cmd: samba_cmds.SambaCommand = samba_cmds.ctdb):
+    cmd_ctdb_check = cmd["nodestatus"]
+    samba_cmds.execute(cmd_ctdb_check)
```

Comment:
When we discussed this before I thought we needed to read the stdout of the ctdb command. If so, we probably want to use a subprocess,  rather than execute, because execute does not fork.
If we don't need to read stdout, that's fine. I just want to understand better  what the expectation is.

Reply:
No. We do not need to parse the output of the 'ctdb nodestatus' command. The return value of the ctdb nodestatus command  indicates the status of the ctdb service. Since the readiness probe depends solely on the return value, it made sense to just use exec() instead of fork() and exec()

Reply:
OK.